zarr/storage.py

```diff
@@ -1994,8 +1994,7 @@ def _append_path_to_prefix(self, path):
         if self.prefix == '':
             return normalize_storage_path(path)
         else:
-            return '/'.join([normalize_storage_path(self.prefix),
-                             normalize_storage_path(path)])
+            return '/'.join([self.prefix, normalize_storage_path(path)])
```

Comment:
So what happens here if prefix='/'?

Reply:
Because the `prefix` is normalized in the constructor, if you pass in `'/'` for the `prefix` it will still be stored as `''` in `self.prefix`

Reply:
Okay great. Thanks.